### DISCUSSION TITLE
B站部分不渲染

### DISCUSSION THREAD

**Discussion by MTYao** [F9y4ng/GreasyFork-Scripts#195] [问答专区(Question / Answer)] [answered]
作者你好，我开了字体重写，用的是自添加的汉仪正圆，发现B站主页有部分内容字体没有被重写。使用是全局设置，其他网站暂未发现此情况。
![Quicker_20230409_180825](https://user-images.githubusercontent.com/35882418/230767068-7b6207b8-53f8-4587-ab42-76516f1b2aff.png)     
![Quicker_20230409_180818](https://user-images.githubusercontent.com/35882418/230767069-1c4d5939-23ce-4cb3-aa45-c1278f8cc64e.png)

**Comment by F9y4ng** [comment, marked as answer]
添加自定义字体，需要添加PostScript Name才能实现全局热替换，参照 #64 将自定义字体参数补充完整。

**Reply by MTYao** [reply]
已解决。多谢解答！

**Comment by F9y4ng** [comment]
非自定义字体未添加PostScript Name造成字体热替换无效的用户，请更新至V2023.04.10.2，重置数据后应用修正后的过滤规则即可。
